crates/loro-internal/src/undo.rs

```diff
@@ -428,6 +458,14 @@ impl UndoManagerInner {
             return;
         }
 
+        if self.group_active {
+            let affected_cids: FxHashSet<_> = event
```

Comment:
since self.group_affected_cids is already a hashset, we don't need to create another set. just iter the list and insert

Reply:
fixed